packages/playwright/tests/test_utils.js

```diff
@@ -364,7 +364,7 @@ export async function createNewFeature(page) {
 
   // Wait until we are on the Feature page.
   await page.waitForURL('**/feature/*');
-  await delay(1500);
```

Comment:
Since you use sl-skeleton as the loading indicator, thoughts on waiting for the skeleton component to actually be gone?

```ts
const skeletonLocator = page.locator('sl-skeleton');
await expect(skeletonLocator).toHaveCount(0);
```

or 

```ts
const skeletonLocator = page.locator('sl-skeleton');
await skeletonLocator.waitFor({ state: 'hidden' });
```

We started doing something similar in [webstatus](https://github.com/GoogleChrome/webstatus.dev/blob/c764df32411c5c0db46e327119d8c1f93cb0a67e/e2e/tests/utils.ts#L61-L65) and it helped out a lot.